README.md

```diff
@@ -16,6 +16,14 @@ yarn start
 
 or use `F5` in Visual Studio Code.
 
+## Package
+
+```
+yarn dist
+```
+
+You find the generated installer in the dist folder.
+
 ## Current state
 
 It launches, initializes OBS Studio and lets you press "Start recording" button. On recording it captures your desktop video, your webcam picture (if available), audio, and microphone to the video file in `videos/` subfolder. Look at main process' console output.
```

Comment:
There is also leftover reference for `videos/` subfolder that should be changed somehow. E.g. like this:

```suggestion
 It launches, initializes OBS Studio and lets you press "Start recording" button. On recording it captures your desktop video, your webcam picture (if available), audio, and microphone to the video file in `Videos` folder in your user profile directory. Look at main process' console output.
```